services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableIndexManagerImpl.java

```diff
@@ -708,6 +712,17 @@ void applyRowChangeToIndex(IdAndVersion idAndVersion, ChangeData<SparseChangeSet
 		applyChangeSetToIndex(idAndVersion, sparseChangeSet, rowChange.getChangeNumber());
 	}
 	
+	void applySearchChangeToIndex(IdAndVersion idAndVersion, ChangeData<SearchChange> loadChangeData) {
+		if (loadChangeData.getChange().isEnabled()) {
+			// This will make sure that the table is properly created
+			updateTableSchema(idAndVersion, false, Collections.emptyList());
+			tableIndexDao.addSearchColumn(idAndVersion);
+		}
+		
+		// set the new max version for the index
+		tableIndexDao.setMaxCurrentCompleteVersionForTable(idAndVersion, loadChangeData.getChangeNumber());

```

Comment:
in here add a column to the state table for the boolean state,

Reply:
Not needed at this stage, will probably add it if needed in the next PR.